### DISCUSSION TITLE
bug: project nvim: `<c-d>` (to delete item from project list) only work once

### DISCUSSION THREAD

**Discussion by erlangparasu** [LazyVim/LazyVim#4310] [Q&A]
extras/util/project
`project.nvim`

bug: 
`i` `<c-d>` (to delete item from project list) only work once

**Comment by dpetka2001** [comment]
Unfortunately, this is a bug in `project.nvim`. There is a [PR](https://github.com/ahmedkhalf/project.nvim/pull/106/files) to fix that, but there hasn't been any movement in the github repo for over a year. I just made a PR #4314 to fix this locally in LazyVim. Not sure if the maintainer accepts it, since ***it is something*** that should be fixed upstream instead.
